### DISCUSSION TITLE
Custom link based on Tags

### DISCUSSION THREAD

**Discussion by BadiaJunkie** [netbox-community/netbox#8240] [Q&A] [answered]
I'm a newby with Netbox and trying to create a custom link which uses a tag assigned to a specific device. The content type i'm using is DCIM>device to show a link in this case to some sharepoint documentation. I am using this in the Link Text template field

```
{% if obj.tags.name == 'SDWAN' %}SDWAN Documentaton{% endif %}
```

but nothing shows up. When i use other fields like obj.rack.name it works fine with the right rackname. But when i look in de REST API information the required fields are filled, what am i not doing right?

**Comment by jeremystretch** [comment, marked as answer]
You're very close! Try this:

```
{% if 'SDWAN' in obj.tags.names %}SDWAN Documentaton{% endif %}
```

**Reply by BadiaJunkie** [reply]
Thanks, now i am getting closer. Now it gives a error _argument of type 'method' is not iterable_.

**Reply by jeremystretch** [reply]
My mistake! It should be `{% if 'SDWAN' in obj.tags.names() %}`. I was thinking in DTL rather than Jinja2.

**Reply by BadiaJunkie** [reply]
Yes,  that worked like a charm!!! Thank you!
